### DISCUSSION TITLE
Lost my github account

### DISCUSSION THREAD

**Discussion by rayan-haroon** [community/community#126399] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was a new user to github and didn't knew much about it. I was just trying to secure to secure my account and enabled 2FA. But I did'nt set up the 2FA. Then also forgot my password and recovery codes. Kingdly, tell me if there is any way to get my account back.

**Comment by davevad93** [comment]
Hi @rayan-haroon ,

I think you should contact GitHub Support and open a ticket [here](https://support.github.com/contact/account), select one between `Two-factor authentication recovery` or `Sign in, verification or other account questions`.

**Comment by ebndev** [comment]
Hi @rayan-haroon,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
